substrate/frame/Cargo.toml

```diff
@@ -7,7 +7,6 @@ license = "Apache-2.0"
 homepage = "paritytech.github.io"
```

Comment:
The check misses this but homepage must be a url.

Reply:
Fixed